enterprise-suite/tests/smoke_prometheus

```diff
@@ -237,17 +248,40 @@ busy_wait app_data
 
 prom_has_data "$app_instances"
 
+# make_monitor "type/name" "metric"
+make_monitor() {
+  curl -XPOST -H "Content-Type: application/json" -H 'Author-Name: Me' -H 'Author-Email: me@lightbend.com' -H 'Message: testing!' \
+    "$ES_MONITOR_API/monitors/$1" \
+    --data @- <<EOF
+    {
```

Comment:
Might want to put that in a file in the `resources` dir.  Or not.

Reply:
oh! good idea